RSDKv5/RSDK/Input/Wii/WiiInputDevice.cpp

```diff
@@ -37,10 +37,8 @@ void RSDK::SKU::InputDeviceWii::UpdateInput() {
             this->stateZ      = (this->buttonMasksWii & 0) != 0;
             this->stateStart  = (this->buttonMasksWii & WPAD_BUTTON_PLUS) != 0;
             this->stateSelect = (this->buttonMasksWii & WPAD_BUTTON_MINUS) != 0;
-            this->stateUp     |= (data->exp.nunchuk.js.pos.y > data->exp.nunchuk.js.center.y + 15) ? 1 : 0;
-            this->stateDown   |= (data->exp.nunchuk.js.pos.y < data->exp.nunchuk.js.center.y - 15) ? 1 : 0;
-            this->stateLeft   |= (data->exp.nunchuk.js.pos.x < data->exp.nunchuk.js.center.x - 15) ? 1 : 0;
-            this->stateRight  |= (data->exp.nunchuk.js.pos.x > data->exp.nunchuk.js.center.x + 15) ? 1 : 0;
+            this->vDelta_L    = (float)(data->exp.nunchuk.js.pos.y - data->exp.nunchuk.js.center.y) / (data->exp.nunchuk.js.max.y / 2);
+            this->hDelta_L    = (float)(data->exp.nunchuk.js.pos.x - data->exp.nunchuk.js.center.x) / (data->exp.nunchuk.js.max.x / 2);
```

Comment:
`(data->exp.nunchuk.js.max.y / 2)` is an integer division, so we might lose precision in the second division as well.
Consider casting every element to `float` individually and divide by `2.0f` instead of `2`.